packages/program/src/script-request.ts

```diff
@@ -38,9 +38,9 @@ export type ScriptResult = {
   receipts: TransactionResultReceipt[];
   scriptResultReceipt: TransactionResultScriptResultReceipt;
   returnReceipt:
-    | TransactionResultReturnReceipt
-    | TransactionResultReturnDataReceipt
-    | TransactionResultRevertReceipt;
+  | TransactionResultReturnReceipt
```

Comment:
I like this way of formatting discriminated unions, the whitespace is unnecessary because the `|` gives a visual distinction.